### PR TITLE
Create bug_report.md

### PR DESCRIPTION
**Pull request by djdanielsson**
### What does this PR do?

adds bug report template

### How should this be tested?

none

### Is there a relevant Issue open for this?

none

### Other Relevant info, PRs, etc

none

### CODE REVIEW

**Comment by djdanielsson** [comment]
I just tweaked a copy of the ansible-lint bug report template, looking for what other content we should add in ours.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Please give some details of what is actually happening.
+Include a [minimum complete verifiable example] with:
+
+- playbook
+
+<!--- Paste verbatim command output between triple backticks -->
+
+```paste below
+
+```
```
Comment (branic) [inline .github/ISSUE_TEMPLATE/bug_report.md:59]: You missed including this:

`[minimum complete verifiable example]: http://stackoverflow.com/help/mcve`

This allows the text on line 61 to be a hyperlink. I think it is good that we include the link.

**Review by branic — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (djdanielsson) [reply .github/ISSUE_TEMPLATE/bug_report.md:59]: thanks, added it back in

**Review by branic — APPROVED** [review]
LGTM

**Review by sean-m-sullivan — APPROVED** [review]
lgtm

**Pull request merged**